Update import location for SQLAlchemy project reorganization.

jtbaker opened the pull request:
For the pending 1.4 SQLAlchemy release, the `_class_resolver` class is moving to the `orm` module [here](https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/orm/clsregistry.py#L317-L354). This PR updates the module import path for one of the dependencies.

mrjoes commented:
Can you add a try/except around the import and use the old import to keep backwards compatibility?

jtbaker commented:
> Can you add a try/except around the import and use the old import to keep backwards compatibility?

Will do, good idea.  👍

jtbaker commented:
@mrjoes I updated the PR with your suggestion and added a few comments about the handling. If you'd rather not have those in the source code, I'm happy to remove.

mrjoes commented:
This is good, thank you!
